### ISSUE TITLE
Issue when image becomes smaller (when zooming out) than viewport

### ISSUE THREAD

**Issue by jonathangerbaud** [alexvasilkov/GestureViews#41]
Hello,

First of all, let me say that your library is amazing!

Now, I'll try to explain the issue precisely. The problem occurs when `setRestrictBounds` is set to `false`.
When I zoom out and the image becomes smaller than its original size, then gestures (rotation, panning) are almost impossible and seems no longer enabled. Only zooming in seems to work (but not as efficiently as usual behavior).

It's like there is some condition that prevents gesture when the image is smaller than its original size.

And it's quite annoying for user experience.

I tried several combinations of settings but nothing seems to fix this issue. 

Here is the actual settings configuration I'm using:

``` java
      source.getController().getSettings()
                .setMaxZoom(5f)
                .setPanEnabled(true)
                .setZoomEnabled(true)
                .setDoubleTapEnabled(false)
                .setRotationEnabled(true)
                .setRestrictRotation(false)
                .setRestrictBounds(false)
                .setOverscrollDistance(0f, 0f)
                .setOverzoomFactor(2f)
//                .setFillViewport(true)
                .setFitMethod(Settings.Fit.OUTSIDE)
                .setGravity(Gravity.CENTER);
```

Maybe I missed a settings, or there is some trick to use to make it work. Any way, if you have any clue, let me know about it.

Thank you in advance for your time!

**Comment by tdekoning** [comment]
I'm experiencing the same issue with the following configuration:
```
gestureView.getController()
        .getSettings()
        .setMaxZoom(3f)
        .setRotationEnabled(true)
        .setRestrictBounds(false)
        .setFitMethod(Settings.Fit.INSIDE)
        .setOverscrollDistance(2f, 2f)
```

**Comment by alexvasilkov** [comment]
Issue is fixed in d2ad654c648e6899cc455d44ae4d8b4cb2cd8576, will be available in upcoming version.
